src/FileStream.cpp

```diff
@@ -876,7 +858,7 @@ static void BaseHttp_Close(TFileStream * pStream)
     if(pStream->Base.Http.hInternet != NULL)

```

Comment:
Given that those Internet functions are no longer used, I am afraid you are working on an old version of StormLib.

Reply:
We do not use these functions either, I only added the comment here for consistency